src/ceph-volume/plugin/zfs/CONTRIBUTING.rst

```diff
@@ -0,0 +1,128 @@
+.. highlight:: shell
```

Comment:
i guess this file comes from an existing template. and i don't think this `CONTRIBUTING.rst` applies to ceph or ceph-volume-zfs. as it's incorrect and misleading.

Reply:
@tchaikov 
As per Alfredo's advise I used a template generator.
Perhaps it is wiser to remove most of this cruft, as in the end it is going to be part of ceph, instead of a separate project.